marlowe-dashboard-client/src/Contract/View.purs

```diff
@@ -414,16 +416,17 @@ renderCurrentStep currentSlot state =
           [ span
               [ classNames [ "text-xl", "font-semibold", "flex-grow" ] ]
               [ text $ "Step " <> show (stepNumber + 1) ]
-          , if contractIsClosed then
-              statusIndicator Nothing "Contract closed" [ "bg-lightgray" ]
-            else
-              statusIndicator (Just Timer) timeoutStr [ "bg-lightgray" ]
+          , case contractIsClosed, isJust pendingTransaction of
+              true, _ -> statusIndicator Nothing "Contract closed" [ "bg-lightgray" ]
+              _, true -> statusIndicator Nothing "Awaiting confirmation" [ "bg-lightgray" ]
+              _, _ -> statusIndicator (Just Timer) timeoutStr [ "bg-lightgray" ]
           ]
       , div [ classNames [ "overflow-y-auto", "px-4", "h-full" ] ]
-          [ case currentTab /\ contractIsClosed of
-              Tasks /\ false -> renderTasks state
-              Tasks /\ true -> renderContractClose
-              Balances /\ _ -> renderBalances state balances
+          [ case currentTab, contractIsClosed, isJust pendingTransaction of
```

Comment:
I'm surprissed (in a good way) about the change of `/\` for `,`... how is that working? what is the data type for `a, b, c`

Reply:
It's just syntactic sugar for case expressions: https://github.com/purescript/documentation/blob/master/language/Syntax.md#case-expressions. I don't know if it's implemented underneath as a list or a tuple or what. :man_shrugging:

Reply:
TIL

Reply:
the answer: a simple `&&` in an if statement... very good, I'll replace the tuple when I see them
![Screen Shot 2021-06-15 at 17 17 10](https://user-images.githubusercontent.com/2634059/122117733-94b14e80-cdfd-11eb-8ba6-4b5c1c28eb4e.png)